force-app/main/default/classes/DMLOperations.cls

```diff
@@ -160,14 +186,24 @@ public with sharing class DMLOperations {
 	 */
 	public static Account upsertAccount(String accountName) {
 		// Query for the Account
-		List<Account> accounts = [SELECT Id FROM Account WHERE Name = :accountName];
+		List<Account> accountsList = [SELECT Id FROM Account WHERE Name = :accountName];
 
-		// if the Account exists, update the description
+        Account upsertedAccount = new Account();
```

Comment:
A simple variable name like `acc` would work better here since this Account hasn't been upserted yet. For that reason, you could even use `accToUpsert`.